app/assets/javascripts/discourse-common/addon/lib/icon-library.js

```diff
@@ -2,6 +2,7 @@ import I18n from "I18n";
 import attributeHook from "discourse-common/lib/attribute-hook";
 import { h } from "virtual-dom";
 import { isDevelopment } from "discourse-common/config/environment";
+import { escapeExpression } from "discourse/lib/utilities";
```

Comment:
`discourse/lib/utilities` is not available in discourse-common (hence the test failure)

Reply:
It doesn't look like there is an alternative in `discourse-common` 😢 Is there a solution better than copying/pasting the method here?

Reply:
The best course of action would be:
1. Move the function to discourse-common
2. Re-export it in `discourse/lib/utilities` to maintain backward compatibility

Reply:
Ok this is done let me know what you think (I did switch from `escapeExpression` to `escape` in `pretty-text` because the former is just a wrapper around the later).